### DISCUSSION TITLE
Meta API updated and unable to add new IP whitelist at Azure NSG

### DISCUSSION THREAD

**Discussion by UrbanovichA** [community/community#66106] [Apps, API and Webhooks]
### Select Topic Area

Question

### Body

Hi Community
Ive got automated solution for whitelist github agents IP addresses that are published at https://api.github.com/meta on Azure NetworkSecurityGroup rule. All works fine
But today my solution failed with error:

_creating/updating Security Rule: (Name "AllowAgents_ipv4" / Network Security Group Name "my_nsg" / Resource Group "my_rg"): network.SecurityRulesClient#CreateOrUpdate: Failure sending request: StatusCode=400 -- Original Error: Code="SecurityRuleInvalidAddressPrefix" Message="Security rule /subscriptions/my_subscribtion/resourceGroups/my_rg/providers/Microsoft.Network/networkSecurityGroups/my_nsg/securityRules/AllowAgents_ipv4 has invalid Address prefix. Value provided: 10.152.0.69/31" Details=[]_

On https://api.github.com/meta those address 10.152.0.69/31 can be found, so ive tried to add it manually to existing or newly created nsg - result is the same, and here is response from Azure GUI
_10.152.0.69/31 is not a valid CIDR block. Use 10.152.0.68/31 instead._
Manually i can add to nsg 10.152.0.**68**/31, but not 10.152.0.**69**/31

Can you please advice if this is correct, or just a mistake at Meta?

thanks

**Comment by LewisJackson1** [comment]
We're also getting this for an AWS WAF IP set. It's not a valid CIDR block.

**Reply by UrbanovichA** [reply]
Interesting how it was added on Git API, and when it will be solved...
